app/javascript/components/SearchPanel.js

```diff
@@ -23,20 +23,24 @@ export const SearchContext = React.createContext(searchContexts.study);
 /**
  * Component for SCP faceted search UI
  *
- * This is the entry point into React code from the traditional JS code
+ * This is an entry point into React code from the traditional JS code
  * See related integration at /app/javascript/packs/application.js
  */
 export default function SearchPanel() {
   // Note: This might become  a Higher-Order Component (HOC).
   // This search component is currently specific to the "Studies" tab, but
   // could possibly also enable search for "Genes" and "Cells" tabs.
 
+  // TODO (SCP-2169): Remove this crude mock data with a prop when
+  // 1) HomePage component is merged and and 2) /search response is updated.
+  const matchingStudies = ['SCP1', 'SCP2'];
```

Comment:
For reference, we're going to call this `matching_accessions` in search API responses.

Reply:
I've added a note clarifying this in 68fc531.

Reply:
Thanks for the note, I've added a comment to that effect in 68fc531.